ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/offers/OfferMapper.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.cmc.ccd.mapper.offers;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.offers.CCDOffer;
+import uk.gov.hmcts.cmc.ccd.mapper.Mapper;
+import uk.gov.hmcts.cmc.domain.models.offers.Offer;
+
+@Component
+public class OfferMapper implements Mapper<CCDOffer, Offer> {
+    @Override
+    public CCDOffer to(Offer offer) {
+        CCDOffer.CCDOfferBuilder builder = CCDOffer.builder();
```

Comment:
this could be inlined into one statement,
CCDOffer.builder()
 .content..
.completionDate...

REQ